packages/core/src/sync-realtime/service.ts

```diff
@@ -748,21 +723,23 @@ export class RealtimeSyncService extends Emittery<RealtimeSyncEvents> {
    * Check if deep re-org occured by comparing remote "finalized" block to local.
    */
   private reconcileDeepReorg = async (latestBlockNumber: number) => {
+    this.common.logger.trace({
+      service: "realtime",
+      msg: `Checking if reorg has occurred before or at block ${this.finalizedBlock.number} (network=${this.network.name})`,
+    });
+
     const remoteFinalizedBlock = await this._eth_getBlockByNumber(
       this.finalizedBlock.number,
     );
 
     if (remoteFinalizedBlock.hash !== this.finalizedBlock.hash) {
       this.emit("deepReorg", {
         detectedAtBlockNumber: latestBlockNumber,
-        minimumDepth: latestBlockNumber - this.blocks[0].number,
       });
 
       this.common.logger.warn({
         service: "realtime",
-        msg: `Unable to reconcile >${
-          latestBlockNumber - this.blocks[0].number
-        }-block reorg (network=${this.network.name})`,
+        msg: `Detected >${this.network.finalityBlockCount} block reorg (network=${this.network.name})`,
```

Comment:
I think we should move this to `logger.error`, and message

`
Detected unrecoverable reorg at block ${this.finalizedBlock.number} with local hash ${this.finalizedBlock.hash} and remote hash ${remoteFinalizedBlock.hash} (network=${this.network.name})
`